hack/lib/build/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This library holds utility functions for building releases.
+
+# os::build::release::check_for_rpms checks that an RPM release has been built
+function os::build::release::check_for_rpms() {
+	if [[ ! -d "${OS_LOCAL_RPMPATH}" || ! -s "${OS_LOCAL_RELEASEPATH}/CHECKSUM" ]]; then
```

Comment:
In my investigation before I had placed a `set -o xtrace` into the build and I saw the CHECKSUM etc get placed down, but some of these files for whatever reason were not getting copied back from the `hack/env` container AFAIR